core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -475,6 +475,7 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
                     )
                         rejectionReasons.add(RejectionReason.ShouldNotBeDisplayed)
 
+                // Shouldn't this be "Enables nuclear weapon_s_"?
```

Comment:
I personally want this to be removed entirely and replaced by "Only available \<when [Apollo Program] has been completed]>" or somesuch

Reply:
Currently we have the nuclear UNITS saying they need the building, and the BUILDING saying it enables the units, with no direct connection, which is suboptimal

Reply:
Agree, though for user clarity having it both ways might be benficial